### DISCUSSION TITLE
May be worth mentioning that a re-install is needed for Zotero 7 beta

### DISCUSSION THREAD

**Discussion by nathancashion** [retorquere/zotero-better-bibtex#2820] [General]
(posted to Zotero forum:)

I've been holding off on using the 7 beta because some add-ons are not yet compatible… or so I thought.

The most essential add-on for my workflow is BetterBibTeX for use with Obsidian (and Raycast, somewhat). Even though I had updated to the latest beta and the latest BBT, it was still showing as incompatible in the Add-Ons pane.
![Screenshot 2024-03-21 at 10 11 13 AM](https://github.com/retorquere/zotero-better-bibtex/assets/67849068/2aa18eba-f028-4018-afa9-936094a126e6)

But I started seeing comments from others that it was working fine.

It took some guessing and head-banging, but eventually, I tried downloading the latest release from the [GitHub repo](https://github.com/retorquere/zotero-better-bibtex) and manually installing it to Z7b.

What do you know, now it works!
![Screenshot 2024-03-21 at 10 27 34 AM](https://github.com/retorquere/zotero-better-bibtex/assets/67849068/30c15084-be6b-45ab-a773-359e36bda7cd)

This was after a lot of searching forums, Reddit, etc. and not once have I read anyone mention needing to re-install the add-on.

It may be worth mentioning somewhere that even though add-ons automatically show as installed (but disabled) in the beta, they should be re-installed.

**Comment by retorquere** [comment]
I'll pass that on to the zotero crew.

**Reply by retorquere** [reply]
I'm curious how you got in this state though. If you upgraded throughout 6,you should have hit a 6/7 compatible version at some point, and then upgrades would proceed as normal.

**Reply by nathancashion** [reply]
Not sure. I first tried Z7b a month or two ago. Judging by the threads here, you had a compatible build last year. I have auto-updates for add-ons turned on in Z6, and I use Zotero every day. I would have thought it would have worked. Just glad to have it working now.

**Reply by retorquere** [reply]
The Zotero team has a solution, I'm rolling that out now.

**Reply by nathancashion** [reply]
Amazing. Thanks!
